src/utils/marketsAndNetworksConfig.ts

```diff
@@ -153,28 +280,20 @@ const providers: { [network: string]: ethersProviders.Provider } = {};
 export const getProvider = (chainId: ChainId): ethersProviders.Provider => {
   if (!providers[chainId]) {
     const config = getNetworkConfig(chainId);
-    const chainProviders: ethersProviders.FallbackProviderConfig[] = [];
+    const chainProviders: string[] = [];
     if (config.privateJsonRPCUrl) {
-      chainProviders.push({
-        provider: new ethersProviders.StaticJsonRpcProvider(config.privateJsonRPCUrl, chainId),
-        priority: 0,
-      });
+      chainProviders.push(config.privateJsonRPCUrl);
     }
     if (config.publicJsonRPCUrl.length) {
-      config.publicJsonRPCUrl.map((rpc, ix) =>
-        chainProviders.push({
-          provider: new ethersProviders.StaticJsonRpcProvider(rpc, chainId),
-          priority: ix + 1,
-        })
-      );
+      config.publicJsonRPCUrl.map((rpc) => chainProviders.push(rpc));
     }
     if (!chainProviders.length) {
       throw new Error(`${chainId} has no jsonRPCUrl configured`);
     }
     if (chainProviders.length === 1) {
-      providers[chainId] = chainProviders[0].provider;
+      providers[chainId] = new StaticJsonRpcBatchProvider(chainProviders[0], chainId);
```

Comment:
I think the issue related to the cypress test has to do with the new `StaticJsonRpcBatchProvider`. In the cypress setup, we're using the ethers `JsonRpcProvider`.

I'm still digging trying to figure out why it's failing.

Reply:
I don't think the tenderly rpcs support batched requests. I swapped out the batch providers for the `StaticJsonRpcProvider` and it worked.

Is the batch provider needed? Or was that just for performance?

Reply:
It was intended to help with performance, but during testing it didn't seem to give any improvement. I changed it back to `StaticJsonRpcProvider` in all cases to keep the logic consistent between production and test cases